spec/models/user_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe User, type: :model do
+  it 'is not valid without a name' do
+    user = User.new(name: nil)
+    expect(user).to_not be_valid
+  end
+
+  it 'is not valid if posts_counter is not an integer greater than or equal to zero' do
+    user = User.new(posts_counter: -1)
```

Comment:
* As mentioned in the [requiremen](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.)t you are to add unit tests for all your models currently no tests for your likes and comments models, Kindly add them